test-network-function/generic/suite.go

```diff
@@ -208,6 +209,10 @@ var _ = ginkgo.Describe(testsKey, func() {
 		for _, containersUnderTest := range containersUnderTest {
 			testDeployments(containersUnderTest.oc.GetPodNamespace())
 		}
+
+		for _, containersUnderTest := range containersUnderTest {
+			testOwner(containersUnderTest.oc.GetPodNamespace(), containersUnderTest.oc.GetPodName())
```

Comment:
Out of interest, what's the reason for not using the existing `oc` context for the container?

Reply:
I need admin access to the cluster and I do not need to execute anything within the pod.